301/CO_fifo.h

```diff
@@ -471,64 +471,61 @@ size_t CO_fifo_readB642a(CO_fifo_t *fifo, char *buf, size_t count, bool_t end);
 
 
 /** Bitfields for status argument from CO_fifo_cpyTok2U8 function and similar */
-typedef enum {
-    /** Bit is set, if command delimiter is reached in src */
-    CO_fifo_st_closed  = 0x01U,
-    /** Bit is set, if copy was partial and more data are available. If unset
-     * and no error, then all data was successfully copied. */
-    CO_fifo_st_partial = 0x02U,
-    /** Bit is set, if no valid token found */
-    CO_fifo_st_errTok  = 0x10U,
-    /** Bit is set, if value is not valid or out of limits */
-    CO_fifo_st_errVal  = 0x20U,
-    /** Bit is set, if destination buffer is to small */
-    CO_fifo_st_errBuf  = 0x40U,
-    /** Bit is set, if internal error */
-    CO_fifo_st_errInt  = 0x80U,
-    /** Bitmask for error bits */
-    CO_fifo_st_errMask = 0xF0U
-} CO_fifo_st;
+
+
+/**
+ * @defgroup uint8_t Bitfields for status argument from CO_fifo_cpyTok2U8 function and similar
+ * @{
+  */
+#define CO_fifo_st_closed   0x01U  /** Bit is set, if command delimiter is reached in src */
```

Comment:
According to doxygen, documentation comments after a macro or variable must begin with `/**<`
for example:
```
#define CO_fifo_st_closed   0x01U  /**< Bit is set, if command delimiter is reached in src */
```

Reply:
fix on commit https://github.com/CANopenNode/CANopenNode/pull/522/commits/11771f889c75c6f9d78e69d1fc0ffe22cf451604